File: Reporting/VB/DevExpress.DemoReports/IListDataSource/Report.vb
Namespace XtraReportsDemos.IListDataSource

    Public Partial Class Report
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()
            DisplayName = ReportNames.HiddenColumns
            Name = DisplayName
        End Sub
    End Class
End Namespace

File: Reporting/CS/DevExpress.DemoReports/SalesSummary/Report.cs
namespace XtraReportsDemos.SalesSummary {
    public partial class Report : DevExpress.XtraReports.UI.XtraReport {
        public Report() {
            InitializeComponent();
            Name = ReportNames.SalesSummaryName;
            DisplayName = ReportNames.SalesSummary;
        }
    }
}
